api/services/video.js

```diff
@@ -0,0 +1,25 @@
+const axios = require('axios').default;
+
+
+async function getVideoFileFromVimeo(video) {
+  if (video !== '') {
+    let parsed = new URL(video)
```

Comment:
This and lines 8, 15 and 17 just need a semi-colon at the end of this line, for consistency with the style of the rest of the API code